bin/localnet/functions.sh

```diff
@@ -27,8 +24,16 @@ setupInfra () {
         exit 1
     fi
 
+    if [[ ${NET_ENV} == "localnet" ]]; then
+        SERVICES="broker geth"
+    fi
+
+    if [[ ${NET_ENV} == "traversal" ]]; then
+        SERVICES="broker geth ipify"
```

Comment:
ipify is so much needed everythere

Reply:
true, tried removing it, but locationOracle depends on maxmind and it does not resolve internal IPs. Probably I can remake e2e testing using public IPs for containers, but thats in separate PR if really required.